src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -92,6 +98,9 @@
     @Option(shortName = "EXT", doc="Append the given file extension to all metric file names (ex. OUTPUT.illumina_lane_metrics.EXT). None if null", optional=true)
     public String FILE_EXTENSION = null;
 
+    @Option(doc = "Boolean the determines if this run is a NovaSeq run or not. (NovaSeq tile metrics files are in cycle 25 directory.", optional = true)
+    public boolean IS_NOVASEQ = false;
```

Comment:
Is there no way to infer this from the files/directories present on disk? Either to generate this setting or assert that the setting matches the data.